src/components/Swap/SwapActionButton/SwapButton.tsx

```diff
@@ -90,9 +94,14 @@ export default function SwapButton({ disabled }: { disabled: boolean }) {
       // Only close the review modal if the swap submitted (ie no-throw).
       setOpen(false)
     } catch (e) {
-      console.error(e) // ignore error
+      throwAsync(
+        new SwapError({
```

Comment:
These errors are not fatal - they should be dismissable. Right now an error through `throwAsync` can only be dismissed through a page reload.

Can you special-case `SwapError` in `ErrorBoundary`?